code/game/turfs/simulated/wall/mineral_walls.dm

```diff
@@ -153,8 +153,14 @@
 	icon = 'icons/turf/walls/snow_wall.dmi'
 	icon_state = "snow"
 	hardness = 80
+	explosion_block = 0
+	slicing_duration = 30
 	sheet_type = /obj/item/stack/sheet/mineral/snow
 	canSmoothWith = null
+	girder_type = null
+	
+/turf/closed/wall/mineral/snow/devastate_wall()
```

Comment:
Because girders are used in almost every wall and are inherited from normal walls and the only difference between a girderless mineral wall and a girdered mineral wall is the girder, a girderless type should probably be a subtype of the mineral wall rather than an intermediate type.